src/aliceVision/keyframe/KeyframeSelector.cpp

```diff
@@ -30,448 +32,383 @@ namespace keyframe {
  */
 int getRandomInt()
 {
-  std::random_device rd;  // will be used to obtain a seed for the random number engine
-  std::mt19937 randomTwEngine(rd()); // standard mersenne_twister_engine seeded with rd()
-  std::uniform_int_distribution<> randomDist(0, std::numeric_limits<int>::max());
-  return randomDist(randomTwEngine);
+    std::random_device rd;             // will be used to obtain a seed for the random number engine
```

Comment:
This will make testing unreliable. It's best to supply the source of randomness as a parameter so that in tests fake implementation can be used.